src/stores/cluster/constants.ts

```diff
@@ -23,6 +23,13 @@ export const CLUSTER_CREATE_SUCCESS = 'CLUSTER_CREATE_SUCCESS';
 export const V5_CLUSTER_CREATE_SUCCESS = 'V5_CLUSTER_CREATE_SUCCESS';
 export const CLUSTER_CREATE_ERROR = 'CLUSTER_CREATE_ERROR';
 
+export const CLUSTER_COMPLETE_CREATION_REQUEST =
```

Comment:
I feel like "COMPLETE" is a newish domain term and overlaps a bit with "batched".

Could we call these actions "BATCHED_CLUSTER_CREATION_*******" ?

Then it's kinda clear what it's about and we don't introduce the new term ("COMPLETE")